### PR TITLE
Add com.ulduzsoft.KchmViewer (kchmviewer-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**KchmViewer** is a reader for CHM (WinHelp) and EPUB files using Qt toolkit with optional KDE support. The main point of kchmviewer is compatibility with non-English CHM files, including most international charsets.

See also:
http://www.ulduzsoft.com/linux/kchmviewer
https://sourceforge.net/projects/kchmviewer
https://github.com/scx/kchmviewer-flatpak
https://github.com/scx/flathub/tree/com.ulduzsoft.KchmViewer

### CODE REVIEW

**Comment by scx** [comment]
Developers have been notified, but not answered yet.

**Comment by scx** [comment]
testbot, build com.ulduzsoft.KchmViewer

**Comment by barthalion** [comment]
Link to upstream issue about participation?

**Comment by scx** [comment]
@barthalion 

> Link to upstream issue about participation?

I sent an e-mail to the upstream developer regarding this.
As you can see, this project is developed on SourceForge.

File: com.ulduzsoft.KchmViewer.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+app-id: com.ulduzsoft.KchmViewer
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.9'
+runtime: org.kde.Platform
+runtime-version: '5.9'
```
Comment (LeandroStanger) [inline com.ulduzsoft.KchmViewer.yaml]: Update runtime to 5.12

**Comment by LeandroStanger** [comment]
bot, build com.ulduzsoft.KchmViewer

**Pull request closed without merging**